.circleci/config.yml

```diff
@@ -65,8 +69,22 @@ jobs:
       MINIKUBE_WANTUPDATENOTIFICATION: false
       MINIKUBE_WANTREPORTERRORPROMPT: false
       SKAFFOLD_VERSION: v0.18.0
+      GO111MODULE: "on"
```

Comment:
Above, don't you want to build this outside the global gopath? In future versions of Go, it wont enable GO modules by default if you build with the project checked out in the gopath.

Reply:
I will have to keep it:
```
circleci@17ecfd2c0929:/go/src/github.com/virtual-kubelet/virtual-kubelet$ go version
go version go1.12.5 linux/amd64
circleci@17ecfd2c0929:/go/src/github.com/virtual-kubelet/virtual-kubelet$ go mod tidy
go: modules disabled inside GOPATH/src by GO111MODULE=auto; see 'go help modules'
```

Reply:
You should move the project to build outside of the global gopath. Basically, if you change working dir to something like /builds or /foo, it should fix this.

Reply:
I'd rather not change how we build just to support modules.
You *can* build outside of gopath and things will just work, or you can use GO111MODULE=on and it will work.
In any case, I think the `Makefile` should set this (`GO111MODULE ?= on`)
As I recall in the future `on` will be the default even when in GOPATH, however even this has been postponed.